Computer History Museum Archive.js

```diff
@@ -0,0 +1,1291 @@
+						"title": "People's Computers, v. 7, no. 3 (Nov-Dec 1978)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 7, no. 4 (Jan-Feb 1979)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 7, no. 5 (Mar-Apr 1979)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 7, no. 6 (May-Jun 1979)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 8, no. 1 (Jul-Aug 1979)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 8, no. 2 (Sep-Oct 1979)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 8, no. 3 (Nov-Dec 1979)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 8, no. 4 (Jan-Feb 1980)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 8, no. 5 (Mar-Apr 1980)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 8, no. 6 (May-Jun 1980)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 9, no. 1 (Jul-Aug 1980)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 9, no. 2 (Sep-Oct 1980)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 9, no. 3 (Nov-Dec 1980)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 9, no. 4 (Jan-Feb 1981)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 9, no. 5 (Mar-Apr 1981)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 9, no. 6 (May-June 1981)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 10, no. 1 (Jul-Aug 1981)"
+					},
+					{
+						"mimeType": "application/pdf",
+						"title": "Recreational Computing, v. 10, no. 2 (Sep-Oct 1981)"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102630889",
+		"items": [
+			{
+				"itemType": "artwork",
+				"title": "Apple IIe",
+				"creators": [],
+				"abstractNote": "Black and white identification photograph of the Apple IIe main terminal including monitor, disk drive and keyboard. appriximately 1/2 inch white border surrounds main image. Background is gray. Computer is sitting on a ledge.",
+				"archive": "Computer History Museum",
+				"archiveLocation": "X2870.2005",
+				"extra": "Catalog Number: 102630889\nDimensions: 8 x 10 in.\nFormat: Photographic print\nCategory: Identification photograph; Publicity photograph\nCredit: Gift of CHM AppleLore",
```

Comment:
- `Dimensions` is `artworkSize`
- `Format` is `medium`

Reply:
Fixed. Thanks!